bundle/edu.gemini.p2checker/src/main/java/edu/gemini/p2checker/rules/gmos/GmosRule.java

```diff
@@ -1850,6 +1855,33 @@ public IConfigMatcher getMatcher() {
     };
 
     /**
+     * REL-2456: We request a Phase II check to give a warning if the DTA-X
+     * position is not a multiple of the y-pixel binning.
+     */
+    private static IConfigRule DTA_X_Y_MULTIPLE_BINNING_RULE = new IConfigRule() {
+        private static final String errMsg = "The DTA X offset (%d) should be multiple of the Y-binning (%d)";
+
```

Comment:
Right, the value of `PREFIX` is hardcoded in the tests. I have to make public a bunch of methods and attributes.

Shall I do it for the rest of the rules, too?